cmd/bosun/sched/sched.go

```diff
@@ -961,6 +972,14 @@ func (s *Schedule) GetIncidentEvents(id uint64) (*Incident, []Event, []Action, e
 	return incident, list, actions, nil
 }
 
+type IncidentStatus struct {
```

Comment:
unexport this type please. The members can stay Uppercase, but the type should be `incidentStatus`.